Description: Add oslo.db to config generator
 Otherwise we miss the oslo.db configuration option in the sample config file.
From: Julien Danjou <julien@danjou.info>
Origin: upstream, https://review.openstack.org/#/c/127543/
Date: Fri, 10 Oct 2014 14:25:58 +0000 (+0200)
X-Git-Url: https://review.openstack.org/gitweb?p=openstack%2Fceilometer.git;a=commitdiff_plain;h=e179a8382c88374c7fefcb8f1295257a740b7668
Bug-Ubuntu: https://launchpad.net/bugs/1379808
Change-Id: I3a70e5da42562081002286d37ba1a200150c8cfc
Last-Update: 2014-10-10

diff --git a/tools/config/oslo.config.generator.rc b/tools/config/oslo.config.generator.rc
index d964585..122e9fe 100644
--- a/tools/config/oslo.config.generator.rc
+++ b/tools/config/oslo.config.generator.rc
@@ -1,2 +1,2 @@
-CEILOMETER_CONFIG_GENERATOR_EXTRA_LIBRARIES=oslo.messaging
+CEILOMETER_CONFIG_GENERATOR_EXTRA_LIBRARIES="oslo.messaging oslo.db"
 CEILOMETER_CONFIG_GENERATOR_EXTRA_MODULES=keystonemiddleware.auth_token
